swim/memberlist_test.go

```diff
@@ -278,6 +313,26 @@ func (s *MemberlistTestSuite) TestCountReachableMembers() {
 	s.Equal(3, reachableMemberCount, "expected 3 reachable members")
 }
 
+func (s *MemberlistTestSuite) TestRemoveMember() {
+	// seed the membership with more members
+	s.m.Update(s.changes)
+
+	var removed bool
+
+	// remove an unknown member
+	removed = s.m.RemoveMember("192.0.2.123:1234")
+	s.Assert().False(removed, "expect to not remove an unknown member")
```

Comment:
`RemoveMember` returns a boolean if the member was in there before, thus if the removed value is false nothing has been removed.